server.js

```diff
@@ -549,11 +571,17 @@ app.post('/submit-network', asyncMiddleware( async (req, res, next) => {
 
             var training_steps = req.body.training_steps ? Number(req.body.training_steps) : null;
 
+            // Add description of the network, e.g. Regular Network / SWA Network / Test Network
+            //
+            var descrption = req.body.description;
```

Comment:
Is `var descrption = req.body.description ? req.body.description : null;` or similar needed or is undefined the same as null for database storage purposes?

Reply:
I've tried this locally running `/submit-network` without `description`

```
curl -F key=1234 -F weights=@file.gz -F training_count=175000 http://localhost:8080/submit-network
```

mongdb result
```
{
   "_id":"5ab9e31f3e4a4978bf187f43",
   "hash":"5c9065484f9f4ae007f09b2591d5410584c1fa58d6b8bad77fc2a070865c08ab",
   "blocks":10,
   "filters":128,
   "ip":"::1",
   "training_count":175000,
   "training_steps":null,
   "description":null
}
```

Reply:
Good enough for me! I'll merge it and launch it sometime tomorrow when I'm not dead tired. :)